Task 2/Task 2.2/Task2.2.1.GameLogic.Interfaces/IEnemy.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._2._1.GameLogic.Interfaces
+{
+    public interface IEnemy
+    {
+        Point Location { get; }
```

Comment:
All these interfaces require he same Location property, probably it makes sense to 'inherit' them from `IGameObject`?